internal/commands/tag/rm.go

```diff
@@ -52,7 +52,11 @@ func newRmCmd(streams command.Streams, hubClient *hub.Client, parent string) *co
 			metrics.Send(parent, rmName)
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runRm(cmd.Context(), streams, hubClient, opts, args[0])
+			err := runRm(cmd.Context(), streams, hubClient, opts, args[0])
+			if err == nil || err == errdef.ErrCanceled {
```

Comment:
![image](https://user-images.githubusercontent.com/31478878/106445209-01dfcb00-647f-11eb-9878-6672e620956f.png)

Reply:
😂
Fixed the 3 places where the `errors.Is` was missing. 
Thanks @silvin-lubecki